sdk/communication/Azure.Communication.Identity/src/autorest.md

```diff
@@ -7,6 +7,7 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5b19c6e69cd2bb9dbe4e5c1237b2c5a175d90ca5/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/a1eee0489c374782a934ec1f093abd16fa7718ca/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-02-22-preview1/CommunicationIdentity.json
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/a1eee0489c374782a934ec1f093abd16fa7718ca/specification/communication/data-plane/Microsoft.CommunicationServicesTurn/preview/2021-02-22-preview1/CommunicationTurn.json
```

Comment:
Did not you merge the swagger files? You should not need to import them both here

Reply:
I did not merge the swagger files. As I mentioned in our offline chat, I created a preview version of the CommunicationIdentity file with the same preview version as that of the TURN Swagger. Since the TURN Swagger was in a separate folder/location, I didn't move it.